ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.10.yaml

```diff
@@ -152,6 +152,19 @@ tests:
     - chain: upi-vsphere-pre
     - ref: cucushift-pre
     workflow: cucushift-e2e
+- as: e2e-azure-cucushift-upi
```

Comment:
Perhaps we need `azurestack` there to identy that ?

Reply:
@liangxia yes, if the tests succeed, i will fix the naming conventions to azurestack

Reply:
Is azurestack the `Azure Stack Hub`?

Reply:
@jhou1 yes, i guess azure upi is relying on Azure Stack Hub.